app/dashboard/static/js/app/view-tests-job-branch-kernel-plan.2020.1.js

```diff
@@ -0,0 +1,597 @@
+/*!
+ * kernelci dashboard.
+ *
+ * Copyright (C) 2020 Collabora Limited
+ * Author: Guillaume Tucker <guillaume.tucker@collabora.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+require([
+    'jquery',
+    'utils/init',
+    'utils/html',
+    'utils/filter',
+    'utils/format',
+    'utils/error',
+    'utils/request',
+    'utils/table',
+    'tables/test',
+    'charts/passpie',
+    'components/test/view',
+    'URI',
+], function($, init, html, filter, format, error, request, table,
+            testTable, pieChart, testView, URI) {
+    'use strict';
+    var gJob;
+    var gBranch;
+    var gKernel;
+    var gPlan;
+    var gResultFilter;
+    var gFileServer;
+    var gPanel;
+
+    setTimeout(function() {
+        document.getElementById('li-test').setAttribute('class', 'active');
+    }, 15);
+
+    function detailsFailed() {
+        html.replaceByClassTxt('loading-content', '?');
+    }
+
+    function updateDetails(results) {
+        var job;
+        var branch;
+        var kernel;
+        var commit;
+        var plan;
```

Comment:
plan is declared and setted, but not is used into the function.

Reply:
Fixed.